src/main/java/com/kodenigaz/desertwind/User.java

```diff
@@ -0,0 +1,20 @@
+package com.kodenigaz.desertwind;
+
+public class User {
+
+  private String userName;
+  private String password;
+
+  public String getUserName() {
```

Comment:
You can use the (project) lombok to avoid writing boilerplate code like getters and setters:
https://projectlombok.org/features/GetterSetter